R/aggregate_dt.R

```diff
@@ -0,0 +1,33 @@
+## Pass in key, format wanted, dataset, colname, 
+
+aggregatedt <- function(.data, method, key, name) {
+  
+  mc <- match.call()
+  dataname <- mc$.data
+  
+  left = paste0(".DATA$", "left")
+  right = paste0(".DATA$", "right")
+    
+  .data <- iNZightTools::separate(.dataset, col, "left", "right", key, "Column")
```

Comment:
Should this be
```suggestion
  .data <- iNZightTools::separate(.data, col, "left", "right", key, "Column")
```
???